client/shared/src/hover/HoverOverlay.scss

```
@@ -19,228 +19,113 @@
     // Make sure content doesn't leak behind border-radius
     overflow: hidden;
     padding-bottom: var(--hover-overlay-vertical-padding);
+}
+
+.hover-overlay__close-button {
```

We [don't use BEM in CSS modules](https://docs.sourcegraph.com/dev/background-information/web/styling#css-modules). Aim only to keep modifiers in BEM styles: 

> Do not use BEM convention in CSS modules. Use short descriptive classes specific only for the corresponding component because CSS modules provide scoping out of the box. It outputs shorter classes that are more readable in the component markup.

So this would become:

```suggestion
.close-button {
```

The same applies to all other selectors changed in this PR. See more examples of global styles to CSS modules conversion [here](https://github.com/sourcegraph/sourcegraph/pulls?q=is%3Apr+author%3Avalerybugakov+is%3Aclosed+codemod+to+css+modules+).